include/godot_cpp/classes/wrapped.hpp

```diff
@@ -42,13 +42,17 @@
 namespace godot {
 
 class ClassDB;
+template <class T>
+class ClassCreator;
```

Comment:
The name `ClassCreator` is so generic, I worry it could end up conflicting with something later. Maybe this could be in the `godot::internal` namespace? Or, maybe the class could be declared inside of the `Wrapped` or `ClassDB` classes?

Reply:
I'd prefer `ClassDB` as the engine also defined the `create` method there, so we are keeping some consistency.